utils/phase_utils.py

```diff
@@ -19,9 +19,11 @@
 
 import aiohttp
 from pydantic import BaseModel, ValidationError
+from pyhooks.types import MiddlemanModelOutput
```

Comment:
FYI, you can use `typing.TYPE_CHECKING` to add [typing-only imports](https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING) (i.e. not actually used at run time), makes for lighter-weight modules (even better when used with `from __future__ import annotations`)